exploration/default-registry-and-mf1-compatibility.md

```diff
@@ -133,58 +141,56 @@ For more information, see [Working with Timezones](https://w3c.github.io/timezon
 
 #### Options
 
-All date/time functions support the following options, which 
-provide high-level control over date/time formats:
-- `calendar` (default is locale-specific)
-  - valid [Unicode Calendar Identifier](https://cldr-smoke.unicode.org/spec/main/ldml/tr35.html#UnicodeCalendarIdentifier)
-- `numberingSystem` (default is locale-specific)
-   - valid [Unicode Number System Identifier](https://cldr-smoke.unicode.org/spec/main/ldml/tr35.html#UnicodeNumberSystemIdentifier)
-- `timeZone` (default is system default time zone or UTC)
-  - valid identifier per [BCP175](https://www.rfc-editor.org/rfc/rfc6557)
-
-In addition to the above high-level options, a function can use either the appropriate
-_style_ options for that function
+A function can use either the appropriate _style_ options for that function
 or can use a collection of _field options_ (but not both) to control the formatted 
 output.
 
+If both are specified, the _style_ option takes precedence.
+
+##### Style Options
+
 The function `:datetime` has these function-specific _style_ options.
 - `dateStyle`
   - `full`
   - `long`
   - `medium`
-  - `short`
+  - `short` (default)
```

Comment:
As discussed previously in https://github.com/unicode-org/message-format-wg/pull/623#discussion_r1480148093, these "default" notes should be dropped.

```suggestion
  - `short`
```